src/components/cardContent/StepTitle.jsx

```diff
@@ -0,0 +1,11 @@
+import './../../index.css';
+import './../../styles/card.css';
+
+const StepTitle = () => {
```

Comment:
Make a Header component that has both the Title and Subtitle.

Header = ({title, subTitle}) => {
  return (<div><h1 className='card-title'>title</h1><h2>subTitle</h2></div>) 
}